cookbooks/aws-parallelcluster-slurm/recipes/install/install_pyxis.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+#
+# Cookbook:: aws-parallelcluster-slurm
+# Recipe:: install_pmix
+#
+# Copyright:: 2013-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+pyxis_version = "0.20.0"
```

Comment:
What is the difference between `default['cluster']['pyxis']['version'] = 'v0.19.0'` and the value you added here.

Lets not hardcode these values in the recipies, but make sure to use attributes.
Using attributes allows the anyone to update the versions using `ExtraChefAttributes`